.env

```diff
@@ -0,0 +1,3 @@
+DEPLOYER_PRIVATE_KEY = "0000000000000000000000"
```

Comment:
Is this file still necessary? Should the thing that fails when this isn't available just fail when this file isn't available or the things from a .env aren't necessary (e.g. does running a local network need a deployer private key?